Resources/doc/index.rst

```diff
@@ -177,12 +188,6 @@ With the standard ``knp_menu.html.twig`` template and your current page being
 
 .. note::
 
-    You only need to extend ``ContainerAware`` if you need the service
-    container to be available via ``$this->container``. You can also implement
```

Comment:
i think the first part of this note is that you don't need to do ContainerAwareInterface unless you actually need the container. this is still true and we should keep that part of the note.